components/NavigationBar.js

```diff
@@ -0,0 +1,124 @@
+  </svg>
+);
+const PlantOverviewIcon = ({ getFillColor }) => (
+  <svg
+    width="45px"
+    height="45px"
+    viewBox="0 0 64 64"
+    xmlns="http://www.w3.org/2000/svg"
+    strokeWidth="0.5"
+    stroke={getFillColor}
+    fill={getFillColor}
+  >
+    <title>Plant Overview Icon</title>
+    <g data-name="Layer 24" id="Layer_24">
+      <path
+        className="cls-1"
+        d="M42.81,20.66a17.31,17.31,0,0,1-5.6-.95l-.94-.36.36-.94c.09-.23,2.25-5.7,7.11-7.87h0c4.86-2.17,10.38-.13,10.61,0l.95.35-.37.94c-.09.24-2.25,5.71-7.11,7.88A12.19,12.19,0,0,1,42.81,20.66ZM39,18.15c1.67.43,5.08,1,8-.31s4.8-4.25,5.6-5.78c-1.68-.43-5.09-1-8,.3h0C41.58,13.69,39.75,16.62,39,18.15Zm5.19-6.7h0Z"
+      />
+      <path
+        className="cls-1"
+        d="M34.61,22.27l-1-.12c-.25,0-6.09-.77-9.39-4.94s-2.68-10-2.66-10.28l.12-1,1,.12c.24,0,6.08.78,9.38,5s2.69,10,2.66,10.27ZM23.5,8.24c0,1.74.28,5.18,2.29,7.72a12.67,12.67,0,0,0,7,4c0-1.73-.27-5.17-2.29-7.72h0C28.48,9.7,25.19,8.64,23.5,8.24Z"
+      />
+      <path
+        className="cls-1"
+        d="M32,36h0a1,1,0,0,1-1-1c0-.6.08-14.75,6.69-16.95a1,1,0,0,1,1.26.63A1,1,0,0,1,38.28,20C33.92,21.4,33,31.31,33,35A1,1,0,0,1,32,36Z"
+      />
+      <path
+        className="cls-1"
+        d="M43.27,60H20.73a1,1,0,0,1-1-.91L18,39.09A1,1,0,0,1,19,38H45a1,1,0,0,1,1,1.09l-1.74,20A1,1,0,0,1,43.27,60ZM21.65,58h20.7l1.56-18H20.09Z"
+      />
+      <path
+        className="cls-1"
+        d="M47,40H17a1,1,0,0,1-1-1V35a1,1,0,0,1,1-1H47a1,1,0,0,1,1,1v4A1,1,0,0,1,47,40ZM18,38H46V36H18Z"
+      />
+    </g>
+  </svg>
+);
+const HomeIcon = ({ getFillColor }) => (
+  <svg
+    width="45px"
+    height="45px"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    strokeWidth="1"
+    fill="none"
+    stroke={getFillColor}
+  >
+    <path
+      stroke-linecap="round"
+      stroke-linejoin="round"
+      d="m2.25 12 8.954-8.955c.44-.439 1.152-.439 1.591 0L21.75 12M4.5 9.75v10.125c0 .621.504 1.125 1.125 1.125H9.75v-4.875c0-.621.504-1.125 1.125-1.125h2.25c.621 0 1.125.504 1.125 1.125V21h4.125c.621 0 1.125-.504 1.125-1.125V9.75M8.25 21h8.25"
+    />
+  </svg>
+);
+
+export default function NavigationBar() {
+  const router = useRouter();
+  const getFillColor = (path) =>
+    router.pathname === path
+      ? "var(--secondary-stroke-color)"
+      : "var(--primary-color)";
+  const getBackgroundColor = (path) =>
+    router.pathname === path ? "var(--secondary-bg-color)" : "";
+
+  return (
+    <StyledFooter>
+      <NavigationList>
```

Comment:
I recommend wrapping the NavigationList in a `nav` element, since that is the element associated with navigation in HTML.

The structure would then look like this:

```js
<StyledFooter>
<nav>
<NavigationList>
...
</NavigationList>
</nav>
<StyledFooter>
```